src/com/facebook/buck/jvm/java/intellij/ij-module.st

```diff
@@ -49,7 +49,14 @@
         %if(generatedFolder.isAndroidResources)% type="java-resource" %endif%/>
     </content>
 }%
+
+%if(!androidFacet.enabled && java_sdk)%
+    <orderEntry type="jdk" jdkName="%java_sdk%" jdkType="JavaSDK" />
```

Comment:
This is purely scoped to java_sdk. If someone wants to use groosdk, it would need to be a separate config key `groovy_sdk` in the intellij block of buckconfig and could be set here.

I would like to get this in to atleast mirror the legacy support and make it module level granularity. The auto detection change should not be part of this change imo